flapjack/ext.py

```diff
@@ -161,7 +161,7 @@ def flatpak_builder(*args, check=None, distcheck=False):
     if check:
         check_index, check_module = next(
             (ix, m) for ix, m in enumerate(manifest['modules'])
-            if m['name'] == check)
+            if isinstance(m, dict) and m['name'] == check)
```

Comment:
Great, this will make sure the element is a module definition, instead of a string with a sub-manifest include.